umh-core/pkg/diagnosis/suite.go

```
@@ -0,0 +1,226 @@
+		outcomes = append(outcomes, runScenario(t, sc, env, f))
+	}
+	return outcomes
+}
+
+// runScenario drives one Scenario and returns the Availability it reached.
+func runScenario[S any](t Table[S], sc Scenario, env Environment, f Feed[S]) Outcome {
+	var sig Signal[S]
+	for _, s := range t.Signals {
+		if s.Name == sc.Signal {
+			sig = s
+			break
```

golangci-lint (2.12.2)

[error] break with no blank line before

(nlreturn)